.github/workflows/build.yaml

```diff
@@ -539,10 +539,10 @@ jobs:
             protonGitRef: ${{ github.event.inputs.protonBranch || 'main' }}
             shard: 2
             shards: 2
-          # CentOS 8: Release Proton-main 1 of 2
+          # CentOS 9: Release Proton-main 1 of 2
           - os: ubuntu-latest
             container: 'centos'
-            containerTag: stream8
+            containerTag: stream9
```

Comment:
@kgiusti, so downstream has dropped rhel 8? If there's still value in being able to run on RHEL 8 then Alma linux should be used, IMO.